vue-app/src/components/BrightIdModal.vue

```diff
@@ -57,13 +57,15 @@ import {
 import { User } from '@/api/user'
 import Transaction from '@/components/Transaction.vue'
 import Loader from '@/components/Loader.vue'
+import Links from '@/components/Links.vue'
 import { LOAD_USER_INFO } from '@/store/action-types'
 import { waitForTransaction } from '@/utils/contracts'
 
 @Component({
   components: {
     Transaction,
     Loader,
+    Links,
```

Comment:
Are we using the Links component? or do we need some cleanup here?

Reply:
At least on this branch, line 16 uses this component.

Reply:
Oh yes, I clicked on the wrong file. It was supposed to be `vue-app/src/components/CallToActionCard.vue`